test/server-test.js

```diff
@@ -44,3 +44,21 @@ lab.test('test server can load vision view engine ', async() => {
   code.expect(success).to.equal(true);
   code.expect(server.registrations.vision).to.not.equal(undefined);
 });
+
+lab.test('loads plugins in order of dependencies ', async() => {
+  const { server } = await hapiconfi(Hapi, { env: 'depend', configPath: `${__dirname}/dependencies` });
+  code.expect(server.settings.app.order.length).to.equal(3); // make sure we load all 3 plugins
+  code.expect(server.settings.app.order).to.equal([1, 2, 3]); // make sure we load all 3 plugins
+});
+
+lab.test('notifies if deprecated _priority field still used', async() => {
+  const oldLog = console.log;
+  const results = [];
+  console.log = (input) => {
+    results.push(input);
+  };
+  const { server } = await hapiconfi(Hapi, { configPath: `${__dirname}/deprecated` });
+  console.log = oldLog;
```

Comment:
Will have to change this to bind to the log event if changed to server.log

Reply:
Yea actually throwing an error might just be the better way since it is kind of a breaking change.